lib/oplog/tail.go

```diff
@@ -441,20 +448,21 @@ func (tailer *Tailer) getStartTime(maxOrdinal int, getTimestampOfLastOplogEntry
 	ts, tsTime, redisErr := redispub.FirstLastProcessedTimestamp(tailer.RedisClients[0], tailer.RedisPrefix, maxOrdinal)
 
 	if redisErr == nil {
-		// we have a last write time, check that it's not too far in the
-		// past
+		// we have a last write time, check that it's not too far in the past
 		if tsTime.After(time.Now().Add(-1 * tailer.MaxCatchUp)) {
-			log.Log.Infof("Found last processed timestamp, resuming oplog tailing from %d", tsTime.Unix())
+			log.Log.Infof("Found last processed timestamp (%d), resuming oplog tailing from %ds ago", tsTime.Unix(), time.Since(tsTime) / time.Second)
 			return ts
 		}
 
-		log.Log.Warnf("Found last processed timestamp, but it was too far in the past (%d). Will start from end of oplog", tsTime.Unix())
+		log.Log.Warnf("Found last processed timestamp (%d), but it was too far in the past (%ds ago). Will start from end of oplog", tsTime.Unix(), time.Since(tsTime) / time.Second)
 	}
 
 	if (redisErr != nil) && (redisErr != redis.Nil) {
 		log.Log.Errorw("Error querying Redis for last processed timestamp. Will start from end of oplog.",
 			"error", redisErr)
 	}
+	
+	metricOplogFailedResume.Inc()
```

Comment:
Two thoughts:
- Maybe we ought to track every resume, and partition by whether it was successful or not?
- It might be nice to make this a histogram, and make the value "how far behind we are in seconds".